src/include/daos_srv/vos.h

```diff
@@ -433,15 +433,20 @@ vos_aggregate(daos_handle_t coh, daos_epoch_range_t *epr,
  * the caller.
  *
  * \param coh		[IN]	Container open handle
- * \param epr		[IN]	The epoch range to discard
- *				keys to discard
+ * \param oidp		[IN]	Optional oid for oid specific discard.  Aggregation should be
+ *				disabled before calling this function in this mode.  This simply
+ *				removes values in the specified epoch range.  It doesn't discard
+ *				any ilog entries. The expectation is the removed data will
+ *				be replaced at the same epoch (or snapshot).  If that is not the
+ *				case, any orphaned entries will be inaccessible.
```

Comment:
Would you remind me what's the use case of this range value discard again? Thanks.

Reply:
@wangdi1 can amend but it's to clear out data on a target that is being reintegrated.  New updates can be happening simultaneously.  The main issue is that evtree doesn't allow overwrite if the extent boundaries don't match exactly.

Reply:
I see, thanks. So if following reintegration failed, the object will be in an inconsistent status (data is gone but ilog is still present)? Is this something we need to worried about?

Reply:
I don't think so because nobody should be reading from the reintegrated target at that point until reintegration succeeds.

Reply:
that is fine for me.